.github/scripts/split-probe.sh

```
@@ -0,0 +1,92 @@
+# local to this repo.
+#
+# Deliberate cross-repo POINTERS are exempt: any line that names the engine
+# repo explicitly (github.com/gridaco/nothing) is a pointer, not a lie.
+# Historical records (HISTORY.md, docs/_history, archived trees) are decay,
+# not lies — excluded. Earned exceptions live in split-probe-allow.txt
+# (path<TAB>reason), asserted non-stale by the P4 half below.
+#
+# Derived from the extraction manifest (the path set that left this repo).
+# ---------------------------------------------------------------------------
+set -uo pipefail
+cd "$(git rev-parse --show-toplevel)"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)